schemas/2016-03-30/Microsoft.Compute.json

```diff
@@ -0,0 +1,938 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-03-30/Microsoft.Compute.json#",
```

Comment:
Please follow the "README.md" of this repo to:
1. add references to root schemas (two deploymentTemplate.json files)
2. add or update test file, at least for the new property added.
3. test the change with node.js scripts.
Please include the new test file in the PR.